src/praisonai/praisonai/cli/features/setup/handler.py

```
@@ -0,0 +1,309 @@
+        config = {
+            "provider": provider,
+            "model": model,
+            "telemetry": True,  # Default enabled in non-interactive mode
+        }
+        
+        self._save_env(praison_home, env_vars, output)
+        self._save_config(praison_home, config, output)
+        
+        output.print_success("Setup complete")
+        output.console.print(f"Configuration saved to {praison_home}")
+        output.console.print(f'Try: [cyan]praisonai "Say hello in one sentence"[/cyan]')
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)